CHANGELOG.md

```diff
@@ -38,6 +38,11 @@
 
 ## [2.0.0] - TBA
 
+#### Breaking changes
+
+- A non-admin-token is provided to specify non-admin rbac user.
```

Comment:
It doesn't seem like this entry is clear, can we be more specific about the exact functional change here, and how it would impact end users?

Reply:
the kong-non-admin-user is a new interface to end user which let them specify non-admin user token, which only has granted permission accessing resource - non-default workspace etc.

Reply:
Maybe the problem is that we're calling this the "non-admin-user", this is meant to be the user the controller uses right? Can we perhaps change the nomenclature to "kong-controller-user"?

Reply:
non-admin-token is passing the token, which represents one user (not an admin) in Kong, so the name is consistent with the data, it is accurate.

Reply:
5ac1d90c5e6aa8319c4fd3169d6e45da4810e7cf